### DISCUSSION TITLE
I'm not receiving messages through fb ads

### DISCUSSION THREAD

**Discussion by luisdanielhj** [chatwoot/chatwoot#5745] [Self hosted]
I am receiving the messages that are made directly to the page, but when a message is made through an advertisement on the page, I do not receive it

{Correction} i am just receiving messages if i the creator of the facebook app send a message to the page, if other profile do this the crm dont receive the message

**Comment by pranavrajs** [comment]
@luisdanielhj Can you please share a screencast describing the issue?

**Reply by lstedile** [reply]
Hi Prana, i have also almost the same issue. On my campaigns (we are a broker that sells proprieties) does not come the image related to what is being offered to the customer on a fb campain. 
I will show you the attached file the message received directly on a WhatsApp number and in the Chatwoot and you will understand it
![A25EAA38-1095-41C2-8B6A-9E880C23085A](https://user-images.githubusercontent.com/94985520/199254579-6c14fbcc-c24e-4119-b8a1-b8ab1d1f6dc8.jpeg)
![EF4DB63F-006A-4F0E-A705-2048BC809004](https://user-images.githubusercontent.com/94985520/199254603-f65a0568-4122-48f2-9177-44b38882bd18.jpeg)
Both are the same message but one came with the image (whats on the phone) and the other one nothing came (Chatwoot). Appreciate your help
Thanks

**Comment by lstedile** [comment]
Attached files 
![D7CC2CAC-79F1-4771-8B62-2DF710DB5E41](https://user-images.githubusercontent.com/94985520/199253547-0dd9dc01-502b-41d8-9324-b8715c0c823f.jpeg)
![0808E5A1-65E5-4A71-A624-95B030FF5D80](https://user-images.githubusercontent.com/94985520/199253557-cc41957e-164e-4b1f-9129-a4bc5cbaf839.jpeg)

**Comment by luisdanielhj** [comment]
I think that I don't have the enough advanced permissions in the fb app, that's why I just recibe messages from the team on the app, not any of my customers
